server/databases.go

```diff
@@ -66,6 +75,36 @@ type rpsResponse struct {
 	RetentionPolicies []rpResponse `json:"retentionPolicies"`
 }
 
+type measurementLinks struct {
+	Self  string `json:"self"`
+	First string `json:"first"`
+	Next  string `json:"next,omitempty"`
+	Prev  string `json:"prev,omitempty"`
+}
+
+func newMeasurementLinks(src int, db string, limit, offset int) measurementLinks {
+	if limit <= 0 {
+		limit = 100
+	}
+
+	base := "/chronograf/v1/sources"
```

Comment:
good question. given the context in the rest of the file, where all of the other pre-existing links responses follow the same pattern, i think it's safe to assume that basepath is handled elsewhere, or else the other links responses would fail as well